src/binary_search.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+int main() {
+	int arr[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20};
+	int low = 0;
+	int high = sizeof(arr) / sizeof(*arr);
+	int item = 10; 
+
+	while(low <= high) {
```

Comment:
Extract the search logic into separate function

Reply:
done